lib/Logger.ts

```diff
@@ -13,7 +13,13 @@ enum Level {
 }
 
 enum Context {
-  GLOBAL,
+  GLOBAL = 'GLOBAL',
+  DB = 'DB',
+  RPC = 'RPC',
+  P2P = 'P2P',
+  ORDERBOOK = 'ORDERBOOK',
+  LND = 'LND',
+  RAIDEN = 'RAIDEN',
 }
 
 const contextFileMap = {
```

Comment:
I think you want to update `contextFileMap` too to have all the new contexts, even though for now they will all point to a single log file.